### ISSUE TITLE
useWaypoint: Failed to use waypoint 

### ISSUE THREAD

**Issue by SirYac** [kolton/d2bot-with-kolbot#735]
18:12:04 (Yac-Hammer)  Error in Countess (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:110, Game: Hammerrun218)
18:12:07 (Yac-Hammer)  Error in Andariel (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:62, Game: Hammerrun218)
18:12:11 (Yac-Hammer)  Error in Travincal (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:109, Game: Hammerrun218)
18:12:15 (Yac-Hammer)  Error in Mephisto (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:109, Game: Hammerrun218)
18:12:19 (Yac-Hammer)  Error in Diablo (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:110, Game: Hammerrun218)
18:12:22 (Yac-Hammer)  Error in Pindleskin (town.js #2180) Town.goToTown: Failed use WP (Area: 1, Ping:78, Game: Hammerrun218)
18:12:26 (Yac-Hammer)  Error in Eldritch (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:78, Game: Hammerrun218)
18:12:29 (Yac-Hammer)  Error in Worldstone (pather.js #992) useWaypoint: Failed to use waypoint (Area: 1, Ping:78, Game: Hammerrun218)

Those problems started lately. Everything started around week ago, without any edit of pather.js file.
As it started it happens very often: at around 30% of all runs.
As you can see my ping isn't high at all when this happens.

When i open game window and pause bot everything goes back to normal (character is "teleported" to its original - server location) and bot continues to run.
I am sure this issue is somehow due to bad-sync between kolbot and battle.net, but is there any way to fix this?
Thank you very much :)

**Comment by sunace** [comment]
same here its very annoying

**Comment by sunace** [comment]
same here very annoying started about a week ago also I though it was network issues but other profiles will still be running with no issues while others are stuck with this error

**Comment by SirYac** [comment]
I've switched to Internet connection and problem seems to be solved. 
I truly have no idea what was going on since both connections have almost no latency and are very high speed.

**Comment by Chrup** [comment]
Same problem. It was started about 2/3 days ago. Min. is few issues per hour, sometimes 10-15 per hour which means for me almost 0 runs. What happens bros? :/

**Comment by DarkHorseDre** [comment]
I saw mine do the same as I was muling - bot was standing near drognan and i got the waypoint errors.  I moved bot manually (without pausing) and he walked to wp but still didn't take it.  then rollback.

it did the same with the errors, then rollback - about 3 times. then it continued.  looks like its down to whatever is causing rollbacks?

**Comment by sunace** [comment]
have tried different networks myself some that I know are for sure no issues and high speed but I still come up with the issues just as in the op my error in the console pings are all under 100

**Comment by DarkHorseDre** [comment]
Yes, my pings are under 40.  We haven't all suddenly experienced local network/internet issues.  What I am saying is that it must be a server side issue.

**Comment by DarkHorseDre** [comment]
Hey @wbeston - I've not studied it but the few times I've intervened I have forced the correction.  When I pause and operate solo I don't get the issue.

I also notice the bot crashes after pausing a lot now, and also fails to pause mid-script more too but not sure if related...

**Comment by jbost1** [comment]
Any way to increase the movement distance once a failed move to the WP has been made? Usually when I manually move off a distance, and let the bot go back to it, everything is fine.
